templates/zerver/create_realm.html

```diff
@@ -7,8 +7,33 @@
 <script type="text/javascript">
 $(function () {
     common.autofocus('#email');
+    init();
 });
+
+function onSubmit (response) {
+    $('#g-recaptcha-response').text(response);  // Add the response to form.
+    document.getElementById('send_confirm').submit();
+}
+
+function init () {
+    $("#send_confirm").find(":submit").click(function () {
+        if (grecaptcha === undefined) {
+          // If reCAPTCHA library was not loaded, stop processing and return.
+          return true;
+        }
+
+        var form = document.getElementById('send_confirm');
+        if (form.checkValidity()) {  // Do client side validation.
+            grecaptcha.execute();
+            // Don't submit form from here. reCAPTCHA will call onSubmit after
+            // validation.
+            return false;
+        }
+    });
+}
 </script>
+<!-- can we include this from Webpack? -->
+<script src='https://www.google.com/recaptcha/api.js' async defer></script>
```

Comment:
I think this JavaScript code needs to go under `static/js/portico`. I'll try tomorrow to move this.